Source/WebKit/WebProcess/Inspector/ServiceWorkerDebuggableProxy.cpp

```diff
@@ -74,6 +82,23 @@ void ServiceWorkerDebuggableProxy::dispatchMessageFromRemote(String&& message)
         webProcessProxy->send(Messages::WebSWContextManagerConnection::DispatchMessageFromInspector(m_identifier, WTFMove(message)), 0);
 }
 
+#if ENABLE(REMOTE_INSPECTOR_SERVICE_WORKER_AUTO_INSPECTION)
+void ServiceWorkerDebuggableProxy::pauseWaitingForAutomaticInspection()
+{
+    RELEASE_LOG(Inspector, "ServiceWorkerDebuggableProxy::pauseWaitingForAutomaticInspection");
+    m_isPausedWaitingForAutomaticInspection = true;
+
+    // No busy-waiting here because the service worker thread already paused itself.
+}
+
+void ServiceWorkerDebuggableProxy::unpauseForResolvedAutomaticInspection()
+{
+    RELEASE_LOG(Inspector, "ServiceWorkerDebuggableProxy::unpauseForResolvedAutomaticInspection");
```

Comment:
I'm not sure we want this to be a RELEASE_LOG, or at least maybe it could be a more readable message.